fontconvert/Makefile

```diff
@@ -1,8 +1,8 @@
 all: fontconvert
 
 CC     = gcc
-CFLAGS = -Wall -I/usr/local/include/freetype2 -I/usr/include/freetype2 -I/usr/include
-LIBS   = -lfreetype
+CFLAGS = -Wall -I/usr/local/include/freetype2 -I/usr/include/freetype2 -I/usr/include -I/opt/homebrew/include/freetype2
+LIBS   = -L/opt/homebrew/lib -lfreetype
```

Comment:
These paths apply to your machine.

The right way to figure out where freetype2 is, would be to have the Makefile use the output of:
`freetype-config --libs` and   `freetype-config --cflags`.